### PR TITLE
PLT-294 Converting preview image modal to use react-bootstrap.

### PR DESCRIPTION
**Pull request by crspeller**
Fixes issue where modal would appear behind background on iOS

### CODE REVIEW

File: web/sass-files/sass/partials/_modal.scss [hunk 1]
```diff
@@ -213,8 +210,15 @@
...
 				@include opacity(0);
+                -webkit-transition: opacity 0.6s;
+                -moz-transition: opacity 0.6s;
+                -o-transition: opacity 0.6s;
+                transition: opacity 0.6s;
 				cursor: pointer;
 				z-index: 9999;
+                &.modal-close--show {
+				    @include opacity(1);
+                }
```
Comment (jwilander) [inline web/sass-files/sass/partials/_modal.scss]: Not to be a stickler, but the CSS indenting looks off :)

**Comment by jwilander** [comment]
+1, other than my one minor comment

File: web/sass-files/sass/partials/_modal.scss (hunk 1 above)
Comment (crspeller) [inline web/sass-files/sass/partials/_modal.scss]: Ugh... it's all indented using tabs. I overwrote my vimrc for the mm directory to only use spaces.  I guess we don't have a style guide for css though.

**Comment by crspeller** [comment]
Fixed CSS indentation to use tabs instead of spaces.

**Comment by coreyhulen** [comment]
+1

**Pull request merged**